Sources/Core/Notifications/Notifications.swift

```diff
@@ -46,6 +46,9 @@ public final class Notifications: NSObject {
         }
     }
     
+    /// Prevent the display of local notifications.
+    public var disableLocalNotifications: Bool = false
```

Comment:
It will be great to add a little bit more info about why it needs. So, basically, if a customer doesn't use push notifications the local notifications will work anyway. Additionally, local notifications won't work if the client disabled to keep a connection in the background `Config.stayConnectedInBackground`.

A minor suggestion :)
```swift
public var isEnabled = true
```

Reply:
Is there any benefit to keeping a connection in background if we're not showing local notifs?

Reply:
Yes, we do that specifically to avoid reconnections and as a side effect, we should show local notifications.

Reply:
We have a use case that we have our own notification system, so we don't want any notification send out of our system.